lib/surface_site_web/live/builtin_components.ex

```diff
@@ -15,8 +15,8 @@ defmodule SurfaceSiteWeb.BuiltinComponents do
   def render(assigns) do
     ~H"""
     <div style="position: relative;">
-      <MobileSidebar/>
-      <div class="sidebar-bg"/>
+      <MobileSidebar />
+      <div class="sidebar-bg" />
       <div class="container is-fullhd">
         <section class="main-content columns">
           <Sidebar />
```

Comment:
Is this the most common approach among other formatters? It looks better without the space, like `<Sidebar/>`, doesn't it? Although I may have written a lot of code with that same space :)

Reply:
Personally I fall on the other side 😄 but I consider it a trivial issue, not a hill I'd die on. (One of the benefits of a code formatter -- remove space for trivial arguments 😄) If people most prefer without the space, I'm happy to remove it.

To your question, Prettier adds the space. Of a random sampling of lower-profile formatters, most leave it however you had it (with or without space), some actually remove the `/`. (This isn't JSX, it's HTML.) A minority add the space.

Reply:
@paulstatezny It's up to you. Without the extra space looks better but, to be honest, I can live just fine if you keep it :)

However, the formatting of void elements makes the code a bit noisy. For instance:

```
<hr />
<br />
```

It looks really odd. 

Since there are just a few of those void elements and the list is pre-defined by the HTML spec, I believe we should handle them differently as expected:

```
<hr>
<br>
```

Reply:
@msaraiva That makes sense.

I will look up those list of elements in the HTML spec to make sure I get the full list.

Where can I find Surface's copy of that, to cross reference? I assume it has a hardcoded list, since the compiler will otherwise throw an error for unclosed tag.

Reply:
Awesome! You can find the list here: https://github.com/msaraiva/surface/blob/9803d90508020f079e4a08192f0df10d8e83b685/lib/surface/compiler/parser.ex#L136-L148